src/Sql/SqlMysql.php

```diff
@@ -123,7 +123,7 @@ public function dbExists()
         return $this->alwaysQuery("SELECT 1;");
     }
 
-    public function listTables()
+    public function listTables($quoted = false)
```

Comment:
@mcdruid I am confused -- `$quoted` is not used at all - it seems? It's optional, so does not break anything, but that feels really weird to look at.

I think that was left over ...

cc @weitzman